### ISSUE TITLE
Add option to hide the "suggestion window" when press key left or right [old pattern]

### ISSUE THREAD

**Issue by cpusam** [microsoft/vscode#110193]
VS code version: 1.50.0, Linux Ubuntu

I would like to suggest an option in the settings to keep the old pattern of hide suggestion.
If this already exist so I can't found it int the settings.
It's very very very tedious and borring the suggestion window not hide like before, and this newer pattern of vscode not help (for me) in the edition. So, please, add some option in the next version to keep the old style of hide the window.

**Comment by ArturoDent** [comment]
Left arrow still closes the suggestion box for me.  Right arrow does nothing.

**Comment by cpusam** [comment]
@ArturoDent Here in my vscode, if I type some word which I have written before, the window only close if:
1-I put the cursor to left until it get out of the word
2-put the curso to right (or until it get out) of the .word
That is very boring, I must stay clicking at the editor just to close the little window.
Sorry if I'm being inconvenient, but I speak about this things just to the devs see any idea  "different".

**Comment by ArturoDent** [comment]
Ah,  I see better.  This is a new requested feature of vscode.  I was assuming you had just typed a trigger character like a <kbd>.</kbd> - if so, an immediate left arrow would close the box.  But in your example you have typed more already and you are right you would have to do more left arrows.

<kbd>Escape</kbd> should close the suggestions box at any time.

**Comment by jrieken** [comment]
You can add a custom keybinding which does that, like so

```

    {
        "key": "right",
        "command": "hideSuggestWidget",
        "when": "suggestWidgetVisible && textInputFocus"
    },
    {
        "key": "left",
        "command": "hideSuggestWidget",
        "when": "suggestWidgetVisible && textInputFocus"
    }
```

**Comment by cpusam** [comment]
@ArturoDent 
> 
> Escape should close the suggestions box at any time.
Man, press escape it's a hard work for me. If the key to hide the suggestion was like "left Ctrl", so I no need look for the keyboard and think press escape to execute what I want. I was so easy just to press right or left to close the window, that for me it's like candy for my brain. "I don't want to think, so, vscode, do the work for me please, I will say what to do" hahaha
I get many lazy thinks like that because of my hability with gamedev, so, it's automatic. 
If God reveals himself in every detail, so, vscode it's a good program for make something like that. Cause, we have so many IDE's and vscode turn the top one, just not because of "it's free".

@jrieken I will test your suggestion, thanks for that.

**Comment by cpusam** [comment]
@jrieken thanks, solved! But, stays the suggestion to create a option to setup this "beforeature"

**Comment by isidorn** [comment]
fyi @menelion @chrisnorman7

**Comment by CameronBieganek** [comment]
I agree with @cpusam that having to hit escape all the time is pretty annoying. I'll use the workaround provided by @jrieken, but I'd rather be able to disable this new feature (while still keeping suggestions on). Perhaps it would be even better to do what PyCharm does:

![image](https://user-images.githubusercontent.com/8310743/100775437-e8f86200-33c8-11eb-8deb-5a0d4ee7c8fd.png)

In PyCharm you can navigate out of the suggestions window by using `Ctrl + Up` or `Ctrl + Down`, which is very handy.
